floater/generators.py

```diff
@@ -161,21 +161,18 @@ def to_mitgcm_format(self, filename, tstart=0, iup=0, mesh='rect', model_grid=No
                 'lon': 1d array of the model grid tracer center longitudes
                 'lat': 1d array of the model grid tracer center latitudes
         """
-        if model_grid is None:
-            if mesh == 'hex':
-                xx, yy = self.get_hexmesh()
-            else:
-                xx, yy = self.get_rectmesh()
-            myx = xx
-
-            ini_times = 1
-
-            # initial positions
-            lon = myx.ravel()
-            lat = yy.ravel()
+
+        if mesh == 'hex':
+            xx, yy = self.get_hexmesh()
         else:
-            #place floats just in ocean
-            lon, lat = np.transpose(self.get_oceancoords(model_grid, mesh))
+            xx, yy = self.get_rectmesh()
+        myx = xx
```

Comment:
This is a very nitpicky comment, but I don't think we need the variable `myx` at all.